src/app/list-of-components/page.tsx

```diff
@@ -0,0 +1,19 @@
+import SiteBanner from "@/components/banner/site-banner";
+import { DocsButton } from "@/components/buttons";
+import RootLayout from "../layout";
+import { Body, H3 } from "@/components/typography";
+
+export default function ListOfComponents() {
+  return (
+    <RootLayout>
```

Comment:
Since you added the `layout.tsx`, this will automatically apply the layout you made in there to any child page. This means you don't need to manually import `RootLayout` here

Reply:
@branberry good call on that, you are correct, it is automatically handled by Next.js. I removed calling it manually from wihtin the page component.